encodings/sequence/src/compute/take.rs

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: Copyright the Vortex contributors
+
+use num_traits::cast::NumCast;
+use vortex_array::arrays::PrimitiveArray;
+use vortex_array::compute::{TakeKernel, TakeKernelAdapter};
+use vortex_array::{Array, ArrayRef, IntoArray, ToCanonical, register_kernel};
+use vortex_dtype::{NativePType, match_each_native_ptype};
+use vortex_error::VortexExpect;
+use vortex_error::VortexResult;
+use vortex_mask::Mask;
+
+use crate::{SequenceArray, SequenceVTable};
+
+impl TakeKernel for SequenceVTable {
+    #[allow(clippy::cast_possible_truncation)]
```

Comment:
I don't think you have a trunctaion here?

Reply:
Removed. Bad copy paste.